src/components/Avatar/Avatar.jsx

```diff
@@ -38,7 +43,8 @@ const Avatar = ({
   bottomRightBadgeProps,
   withoutBorder,
   customSize,
-  customBackgroundColor
+  customBackgroundColor,
```

Comment:
this prop should not be here, if it for making all the avatar be in the same color let's talk about it and find other solution

Reply:
I haven't add this prop, not sure where it comes from, probably from merge with master